frontend/views/components/Tooltip.vue

```diff
@@ -91,19 +144,48 @@ export default {
         document.body.appendChild(el)
 
         const $this = vnode.context // Vue component instance
-
         if (!$this.tooltip) {
           $this.tooltip = el.getBoundingClientRect()
         }
 
+        if (!$this.tooltipWidth) {
+          if ($this.$slots.tooltip) {
+            const elm = $this.$slots.tooltip[0].elm
+            if (elm.offsetWidth) {
+              $this.tooltipWidth = elm.offsetWidth
+              $this.tooltipHeight = elm.offsetHeight
+            }
+          } else {
+            const elm = el.getBoundingClientRect()
+            $this.tooltipWidth = elm.width
+            $this.tooltipHeight = elm.height
+          }
+        }
+
         // That way the adjustPosition() method can have the same logic
         // applied in every tooltip as expected
         $this.adjustPosition()
+        window.addEventListener('resize', $this.adjustPosition)
+
+        if (bindings.value && bindings.value.manual) {
+          document.addEventListener('keydown', e => $this.trapFocus(e, el))
+          window.addEventListener('keyup', $this.handleKeyUp)
+          $this.lastFocus = document.activeElement
+          $this.focusEl(el)
+        }
       },
-      unbind (el) {
+      unbind (el, bindings, vnode) {
+        const $this = vnode.context
         if (el.parentNode) {
           el.parentNode.removeChild(el)
         }
+        if (bindings.value && bindings.value.manual) {
+          document.removeEventListener('keydown', e => $this.trapFocus(e, el))
```

Comment:
My understanding is that `removeEventListener` must be called on the exact saved instance of a function.

Here it's being called on a new instance of a function, `e => $this.trapFocus(e, el)`, instead of a reference to an existing instance. This will result in a memory leak so it must be fixed.